docs/api/portable-stories-jest.md

```diff
@@ -0,0 +1,317 @@
+
+When you want to reuse a story in a different environment, however, it's crucial to understand that all these steps make a story. The portable stories API provides you with the mechanism to recreate that story pipeline in your external environment:
+
+### 1. Apply project-level annotations
+
+[Annotations](#annotations) come from the story itself, that story's component, and the project. The project-level annotatations are those defined in your `.storybook/preview.js` file and by addons you're using. In portable stories, these annotations are not applied automatically—you must apply them yourself.
+
+👉 For this, you use the [`setProjectAnnotations`](#setprojectannotations) API.
+
+### 2. Prepare
+
+The story is prepared by running [`composeStories`](#composestories) or [`composeStory`](#composestory). You do not need to do anything for this step.
+
+### 3. Load
+
+**(optional)**
+
+Stories can prepare data they need (e.g. setting up some mocks or fetching data) before rendering by defining [loaders](../writing-stories/loaders.md). In portable stories, the loaders are not applied automatically—you have to apply them yourself.
+
+👉 For this, you use the [`composeStories`](#composestories) or [`composeStory`](#composestory) API. The composed story will return a `load` method to be called **before** it is rendered.
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'react/portable-stories-jest-with-loaders.ts.mdx',
+    'vue/portable-stories-jest-with-loaders.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+### 4. Render
+
+At this point, the story has been prepared and can be rendered. You pass it into the
+
+The story has been prepared and can be rendered. To render, you pass it into the rendering mechanism of your choice (e.g. Testing Library render function, Vue test utils mount function, etc).
+
+👉 For this, you use the [`composeStories`](#composestories) or [`composeStory`](#composestory) API. The composed Story is a renderable component that can be passed to your rendering mechanism.
+
+### 5. Play
+
+**(optional)**
+
+Finally, stories can define a [play function](../essentials/interactions.md#play-function-for-interactions) to interact with the story and assert on details after it has rendered. In portable stories, the play function does not run automatically—you have to call it yourself.
+
+👉 For this, you use the [`composeStories`](#composestories) or [`composeStory`](#composestory) API. The composed Story will return a `play` method to be called **after** it has rendered.
+
+The play function needs a `canvasElement`, which should be passed by you. A `canvasElement` is the HTML element which wraps your component. Each testing utility provides different ways to retrieve such element, but here's how to do it with Testing Library:
```

Comment:
@yannbf we need to update this now that `canvasElement` is optional/maybe never needed(?)

Reply:
Sure thing!